R/sort_linter.R

```diff
@@ -69,18 +69,18 @@
 #' @seealso [linters] for a complete list of linters available in lintr.
 #' @export
 sort_linter <- function() {
-  order_xpath <- "
+  non_keyword_arg <- "expr[not(preceding-sibling::*[1][self::EQ_SUB])]"
```

Comment:
aka `positional_arg`?

Reply:
Yes... I do think `non_keyword` maps closer to the XPath using `not()` and referencing `EQ_SUB` though

Reply:
okay